docs/notebooks/2.3 Dust Soiling with Kimber.ipynb

```diff
@@ -0,0 +1,398 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "43cd9359-32f3-455a-b9a2-3f419adacc32",
+   "metadata": {},
+   "source": [
+    "July 2024 has been an exceptional month in terms of irradiance anomalies in North America. In one of [Solcast's weekly PVMag publications](https://www.pv-magazine.com/2024/08/02/aerosols-from-wildfires-slash-north-american-solar-production/) we have highlighted how irradiance was severely impacted in the north west of  the continent by aerosols deriving from the raging Canadian wildfires.\n",
+    "\n",
+    "In addition to the reduced irradiance from the aerosol's absorption, another factor to take into account into the overall impact of these aerosols on solar generation is dust soilig as the accumulated particles block or scatter incident light, further reducing power output.g.\n",
+    "\n",
+    "In this tutorial we are going to see how we can model aerosols using PVLib and Solcast's weather data while trying to see the impact of these aerosols on a site that may have been affected by these wildfires.\n",
```

Comment:
It looks like everywhere on their doco, https://pvlib-python.readthedocs.io/en/stable/, "pvlib" is all lower case.  Also, I would expand it a little and say "using the pvlib python toolbox"